docs/persistence.rst

```diff
@@ -0,0 +1,80 @@
+.. _persistence:
+
+Secure persistence with skops
+=============================
+
+.. warning::
+
+   This feature is very early in development, which means that the API is
+   unstable and that it is **not secure** for the moment. Therefore, the same
+   caution as for pickle should be applied: Don't load from sources that you
+   don't trust. In the future, more security will be added.
+
+Skops offers a way to save and load sklearn models without using :mod:`pickle`.
+Pickle is the standard serialization format for sklearn and for Python in
+general. It has the big advantage that it can be used for almost all Python code
+but this flexibility also means that it's inherently insecure. As the Python
+docs say:
+
+    The pickle module is not secure. Only unpickle data you trust. It is
+    possible to construct malicious pickle data which will execute arbitrary
+    code during unpickling. Never unpickle data that could have come from an
+    untrusted source, or that could have been tampered with.
+
+With the :func:`skops.io.save` and :func:`skops.io.load` functions, we aim at
+providing a secure method to load and save sklearn code. In contrast to pickle,
+these functions cannot be used to save arbitrary Python code, but they bypass
+pickle and are thus more secure.
+
+Usage
+-----
+
+Using :func:`skops.io.save` and :func:`skops.io.load` is quite simple. Below is
```

Comment:
I'd personally avoid using a "value" word like `simple` or `easy` since it is relative to a user's experience and can make the tone not as friendly for beginners. I think the code snippet below already does a great job of showing how simple it is without having us tell the user it is simple :)